src/main/java/mate/academy/internetshop/lib/Injector.java

```diff
@@ -0,0 +1,60 @@
+package mate.academy.internetshop.lib;
+
+import java.lang.reflect.Field;
+import mate.academy.internetshop.Main;
+import mate.academy.internetshop.dao.BucketDao;
+import mate.academy.internetshop.dao.ItemDao;
+import mate.academy.internetshop.dao.OrderDao;
+import mate.academy.internetshop.dao.UserDao;
+import mate.academy.internetshop.factory.Factory;
+import mate.academy.internetshop.lib.annotation.Inject;
+import mate.academy.internetshop.service.BucketService;
+import mate.academy.internetshop.service.ItemService;
+import mate.academy.internetshop.service.OrderService;
+import mate.academy.internetshop.service.UserService;
+import mate.academy.internetshop.service.impl.BucketServiceImpl;
+import mate.academy.internetshop.service.impl.ItemServiceImpl;
+import mate.academy.internetshop.service.impl.OrderServiceImpl;
+import mate.academy.internetshop.service.impl.UserServiceImpl;
+
+public class Injector {
+    public static void injectDependency() throws IllegalAccessException {
+        Class[] classes = new Class[]{
+            BucketServiceImpl.class, ItemServiceImpl.class, OrderServiceImpl.class,
+            UserServiceImpl.class, Main.class
+        };
+
+        for (Class clazz : classes) {
+            for (Field field : clazz.getDeclaredFields()) {
```

Comment:
we can create two method:
first of them will get as parameter class or fields and make inject
second will call first method in the `for loop` for all classes or fields